Do not use volumeMode: block in LocalVolume object (#28680)
openshift/openshift-docs: modules/persistent-storage-local-create-cr.adoc

<6> The path containing a list of local storage devices to choose from.
<7> Replace this value with your actual local disks filepath to the `LocalVolume` resource, such as `/dev/xvdg`. PVs are created for these local disks when the provisioner is deployed successfully.
+
[NOTE]
====

> A raw block volume (`volumeMode: block`) is not formatted with a filesystem. You should use this mode only if any application running on the pod can use raw block devices.

Reviewer: ```suggestion
A raw block volume, described as `volumeMode: block`,  is not formatted with a file system. Use this mode only if all applications running on the pod can use raw block devices.
```

> ✅  file system noun
Write as shown: two words.

defined, then the Local Storage Operator will attempt to find matching disks
on all available nodes.
<3> The volume mode, either `Filesystem` or `Block`, defining the type of the
local volumes. 
<4> The file system that is created when the local volume is mounted for the
first time.
<5> The path containing a list of local storage devices to choose from.
<6> Replace this value with your actual local disks filepath to the LocalVolume resource, such as `/dev/xvdg`. PVs are created for these local disks when the provisioner is deployed successfully.
+
[NOTE]
====
If you are creating a local volume for cluster logging or {product-title} Monitoring, do not use a raw block volume (`volumeMode: block`) because Elasticsearch cannot use raw block volumes.
====

> +

Reviewer 2: This document is not specific to elasticsearch and hence we should rather do something bit more generic and expand the meaning of `Block` volume mode.  For example we could say:

>  `volumeMode: Block` means the volume will not be formatted with a filesystem and will remain a raw block device. You should use this mode only if your application within the pod can use raw block devices.
